Biomass_speciesData.R

```diff
@@ -77,6 +77,17 @@ defineModule(sim, list(
                               "Defaults to the using the Canadian Forestry Service, National Forest Inventory,",
                               "kNN-derived stand biomass map."),
                  sourceURL = ""),
+    expectsInput("rawBiomassMap", "RasterLayer",
+                 desc = paste("total biomass raster layer in study area. Only used to create `rasterToMatchLarge`",
+                              "if necessary. Defaults to the Canadian Forestry Service, National Forest Inventory,",
+                              "kNN-derived total aboveground biomass map from 2001 (in tonnes/ha), unless",
+                              "'dataYear' != 2001. See ",
+                              "https://open.canada.ca/data/en/dataset/ec9e2659-1c29-4ddb-87a2-6aced147a990",
+                              "for metadata."),
+                 sourceURL = paste0("http://ftp.maps.canada.ca/pub/nrcan_rncan/Forests_Foret/",
+                                    "canada-forests-attributes_attributs-forests-canada/",
+                                    "2001-attributes_attributs-2001/",
+                                    "NFI_MODIS250m_2001_kNN_Structure_Biomass_TotalLiveAboveGround_v1.tif")),
```

Comment:
i wouldn't provide sourceURL here since it's not used -- we hardcode the url based on dataYear